package.json

```diff
@@ -10,7 +10,8 @@
     "build": "webpack --config webpack.prod.js",
```

Comment:
Could you add a `&& npm run test` to the end of the `build` command so that the tests get run by default when building the client? You could also add a script like `build:notests` for one that skips them during active development when tests are known to be broken.